### DISCUSSION TITLE
Path to Models not working in new Colab?

### DISCUSSION THREAD

**Discussion by OrganicBeej** [TheLastBen/fast-stable-diffusion#2424] [Q&A] [answered]
Hi :)

I've just used the latest Colab Notebook and added the same path to an older directory that have all my Models there, but it's not working?

`FileNotFoundError: No checkpoints found. When searching for checkpoints, looked at:
 - file /content/gdrive/MyDrive/sd/stable-diffusion-webui/model.ckpt
 - directory /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion
 - directory /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/LoraCan't run without a checkpoint. Find and place a .ckpt or .safetensors file into any of those locations.`
 
 This is obviously the main directory for the latest update but it's simply not finding my other path which has worked for many weeks now following updates/new installs which is this:
 
  `/content/gdrive/MyDrive/sd_OLD_July_24/stable-diffusion-webui/models/Stable-diffusion/`
 
 Any advice appreciated. Also, there is a new path to LoRA now. I added the Lora directory (same as above but with just Lora as end directory) and it added LoRA's to the 'Checkpoints' dropdown list?
 
 Bit confused . . 🤷🏼‍♂️

**Comment by TheLastBen** [comment, marked as answer]
it's not path to lora, it's a cell to download loras from links

as for the models path, make sure the path is correct and contains the models

**Reply by OrganicBeej** [reply]
I will double-check but I am sure it is correct as it worked before. Prob user-error then :)
Thanks for your reply, appreciated
